synapse/res/templates/notif.txt

```diff
@@ -10,6 +13,10 @@
 {{ message.body_text_plain }}
 {% elif message.msgtype == "m.file" %}
 {{ message.body_text_plain }}
+{% elif message.msgtype == "m.room.encrypted" %}
```

Comment:
I don't think a `m.room.message`'s `msgtype` would ever be `m.room.encrypted`: https://matrix.org/docs/spec/client_server/r0.6.4#m-room-message-msgtypes

Reply:
Yes I failed to remove this line after realizing that. See higher up in this file where it checks `message.event_type`.